conformance/operator_test.go

```diff
@@ -290,14 +292,15 @@ var _ = Describe("operator", func() {
 				stdout, stderr, err := pod.ExecCommand(clients, podObj, "ip", "addr", "show", "dev", "net1")
 				Expect(err).ToNot(HaveOccurred())
 				Expect(stderr).To(Equal(""))
-				podMac := getMacFrom(stdout)
+				vfID := findVF(intf.Name, numVfs, debugPod)
```

Comment:
This seems extremely fragile. Having leaked pods around from other tests (this suite might not be executed alone) would result in flaky tests that will hunt us later on.

If we don't find a better way to connect the pods' interface with the link on the host, can we try to make this a bit more robust by:
- checking that there is exactly one vfs that is missing from the host
- -checking that before creating the pod, no vfs are missing from the host